### PR TITLE
Discord Birthday Event

### PR DESCRIPTION
**Pull request by kaosj314**
happy (early) birthday discord!

### CODE REVIEW

**Comment by quaackk** [comment]
did you intend for all of these to be sticky?

**Comment by kaosj314** [comment]
shoot you're right

File: src/data/data.json [hunk 1]
```diff
@@ -340,6 +340,24 @@
         4096,
         500000,
         true
+      ], #Happy bday Discord
```
Comment (Iapetus-11) [inline src/data/data.json]: Comments don't work in json

**Review by Iapetus-11 — CHANGES_REQUESTED** [review]

**Comment by kaosj314** [comment]
I have the dumb

**Review by Iapetus-11 — APPROVED** [review]

**Pull request merged**